api/src/main/resources/workbench.yaml

```diff
@@ -1250,6 +1296,41 @@ definitions:
         operator: "BETWEEN"
         operands: ["18", "35"]
 
+  ParticipantListResponse:
+    type: object
+    required:
+      - items
+    properties:
+      items:
+        type: "array"
+        items:
+          $ref: "#/definitions/Participant"
+
+  Participant:
+    type: object
+    required:
+      - cohortId
```

Comment:
Returning cohort ID and CDR version ID for every participant in the list is unnecessary? Maybe we should just use participant ID and status.

Reply:
I removed cohort ID and CDR version ID from the response org.pmiops.workbench.model.Participant, but will still exist in the org.pmiops.workbench.db.model.Participant for persistence.